Filamento.py

```diff
@@ -0,0 +1,76 @@
+from flask import Flask, render_template, request, redirect, session, flash, url_for
+
+class Filamento:
+    def __init__(self, cor, fornecedor, descricao, qtd):
+        self.cor = cor
+        self.fornecedor = fornecedor
+        self.descricao = descricao      
+        self.qtd = qtd 
+
+filamento1 = Filamento('Vermelho','Volt','Vermelho','1000')
+filamento2 = Filamento('Azul piscina','National','Azul tiffany','500')
+filamento3 = Filamento('Branco','National','Branco gelo','300')
+lista = [filamento1 ,filamento2 ,filamento3]       
+ 
+class Usuario:
+    def __init__ (self, nome, nickname, senha):
+        self.nome = nome
+        self.nickname = nickname
+        self.senha = senha
+       
+usuario1 = Usuario('Luiza', 'Lu', '1234')
+usuario2 = Usuario('Leone', 'Leo', '1235')
+usuario3 = Usuario('Laura', 'Laurita', '1236')
```

Comment:
How about to unify these users in one list?
```suggestion
list_users = []
list_users.append(Usuario('Luiza', 'Lu', '1234'))
list_users.append(Usuario('Leone', 'Leo', '1235'))
list_users.append(Usuario('Laura', 'Laurita', '1236'))
```